### PR TITLE
Add make targets for deploying

### PR DESCRIPTION
**Pull request by aleksanb**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,3 +20,11 @@ copy-travis-files:
 .PHONY:
 test:
 	bundle exec rubocop -R -D
+
+.PHONY:
```
Comment (sigvef) [inline Makefile]: Makefile phony syntax looks like this: `.PHONY: deploy-production` :)

Comment (aleksanb) [inline Makefile]: Huh! So has make been creating secret shadow files all along or something?
The command still works even though phony is incorrect.

Comment (sigvef) [inline Makefile]: No, make won't make files based on target names automatically, but if someone were to manually create a file with the same name as a target (e.g. `test`) then make will stop making that target.

**Comment by aleksanb** [comment]
Ping.

**Pull request merged**